.github/workflows/render-all.yml

```diff
@@ -78,51 +78,32 @@ jobs:
           git commit -m 'Render bookdown' || echo "No changes to commit"
           git push origin main || echo "No changes to push"
 
-  render-coursera:
-    name: Render Coursera
-    needs: [yaml-check, render-bookdown]
+  render-tocless:
```

Comment:
This is a new section that will run before Leanpub or Coursera if either is set to `yes`

Reply:
I wonder if it makes sense to call this something like "Start Leanpub/Coursera render" and then have the jobs below be "Finish Leanpub" and "Finish Coursera"